gateway/metrics.go

```diff
@@ -294,7 +295,20 @@ func newHistogramMetric(name string, help string) *prometheus.HistogramVec {
 	return histogramMetric
 }
 
+var tracer = otel.Tracer("boxo/gateway")
+
 // spanTrace starts a new span using the standard IPFS tracing conventions.
-func spanTrace(ctx context.Context, spanName string, opts ...trace.SpanStartOption) (context.Context, trace.Span) {
-	return otel.Tracer("boxo").Start(ctx, fmt.Sprintf("%s.%s", " Gateway", spanName), opts...)
+func spanTrace(ctx context.Context, name string, opts ...trace.SpanStartOption) (context.Context, trace.Span) {
+	return tracer.Start(ctx, name, opts...)
```

Comment:
I simplified this here. If we need to use the previous nomenclature I'm happy to change it.